.github/workflows/update-spec.yaml

```diff
@@ -0,0 +1,94 @@
+name: Update latest specification document in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'spec/asyncapi.md'
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest spec
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v2
+        with:
+          path: spec
+          ref: ${{ github.event.release.target_commitish }}
+      - name: Checkout Another repository
+        uses: actions/checkout@v2
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Get latest release
+        id: latest_version
+        uses: abatilo/release-info-action@v1.3.0
+        with:
+          owner: asyncapi
+          repo: spec
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-spec-${{ steps.latest_version.outputs.latest_tag }}
+      - name: Check for previous spec file and remove it
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ env.GITHUB_TOKEN }}
+          script: |
+            const fs = require("fs");
+
+            const specFiles = fs.readdirSync("./website/pages/docs/specifications");
+
+            const latestRelease = `${{ steps.latest_version.outputs.latest_tag }}`;
+
+            for (const filename of specFiles) {
+              if (filename.startsWith(latestRelease)) {
+                fs.unlinkSync(`./website/pages/docs/specifications/${filename}`);
+              }
+            }
+      - name: Copy Spec file from Current Repo to Another
+        working-directory: ./website
+        run: |
+          cp ../spec/spec/asyncapi.md ./pages/docs/specifications/${{ steps.latest_version.outputs.latest_tag }}.md
+      - name: Remove Table of Contents from Spec
```

Comment:
we should move it to a separate js file under `.github` repo and reuse it instead of duplicating it. Loo at https://github.com/actions/github-script#run-a-separate-file

Reply:
> under .`github` repo

I guess that you had in mind `.github` folder 😄  This workflow is only used in current repo.

Reply:
Moved :)

Reply:
yes, I meant dir not repo 😉